PYME/recipes/recipe.py

```diff
@@ -198,18 +198,18 @@ def execute(self, progress_callback=None, **kwargs):
         #remove anything which is downstream from changed inputs
         
         try:
-            print('recipe.execute()')
+            #print('recipe.execute()')  # possibly change these into logging/debug messages
```

Comment:
This one in particular is quite useful when working out why a recipe doesn't run in PYMEVis/PYMEImage